### PR TITLE
Part of #18714 and  #17670: Replacing deprecated constants

### PR DESCRIPTION
**Pull request by dhruvv173**
## Explanation

- This PR partially fixes following issues
- #18714 
- #17670 

Made changes to follow files

1. `ui/components/app/snaps/copyable/copyable.js`
2. `ui/components/app/snaps/snap-authorship-expanded/snap-authorship-expanded.js`
3. `ui/components/app/snaps/snap-content-footer/snap-content-footer.js`
4. `ui/components/app/snaps/snap-version/snap-version.js`

#### Created story for `Copyable` 
image
 On clicking the icon
image
image

#### Created story for `SnapAuthorshipExpanded`
image
image
image
image

### Before
`snap-content-footer`
image
image

`snap-version`
image
image
image

### After

`snap-content-footer`
image
image

`snap-version`
image
image
image

Covered `jest` test for `snap-version`
image

## Manual Testing Steps

1. Pull this branch
2. Run `yarn storybook`
3. Check following stories: `SnapContentFooter`, `SnapVersion`, `SnapAuthorshipExpanded`, `Copyable`

## Pre-merge author checklist

- [x] I've clearly explained:
  - [x] What problem this PR is solving
  - [x] How this problem was solved
  - [x] How reviewers can test my changes
- [ ] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

**Comment by dhruvv173** [comment]
hey @georgewrmarshall could you please review this PR?
Also, would you like me to create stories for the missing components in the PR? They seem fairly easy.
I didn't create one beforehand as you mentioned its not one of the priorities right now.
Thanks!

**Review by georgewrmarshall — COMMENTED** [review]
Hey @dhruvv173, this is looking great! I think all these components are current and you can go ahead and create stories if they are missing for these components. Thanks!

File: ui/components/app/snaps/copyable/copyable.stories.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Copyable } from './copyable';
+
+export default {
+  title: 'Components/App/Snaps/Copyable',
+
```
Comment (georgewrmarshall) [inline ui/components/app/snaps/copyable/copyable.stories.js:6]: nit: remove line break

File: ui/components/app/snaps/snap-version/snap-version.js [hunk 2]
```diff
@@ -4,57 +4,43 @@ import {
...
+        </>
+      }
+      labelProps={{
+        display: Display.Flex,
+        flexDirection: FlexDirection.Row,
+        alignItems: AlignItems.center,
+      }}
+    />
   );
 };
```
Comment (georgewrmarshall) [inline ui/components/app/snaps/snap-version/snap-version.js:46]: Screenshot 2023-06-13 at 3 59 35 PM
Screenshot 2023-06-13 at 3 59 59 PM

**Review by georgewrmarshall — DISMISSED** [review]
Fantastic work! Made some small updates to reduce deprecated components and update with existing DS components

**Review by georgewrmarshall — APPROVED** [review]

**Review by garrettbear — APPROVED** [review]
thanks @dhruvv173 and thanks @georgewrmarshall helping out the last few items on this PR looks good

**Comment by georgewrmarshall** [comment]
@FrederikBolding @eriknson this one also needs a snaps engineers eyes please 🙏

**Comment by georgewrmarshall** [comment]
Hey @dhruvv173, could you please rebase this PR?

**Pull request closed without merging**